src/js/live-tracker.js

```diff
@@ -3,12 +3,32 @@ import mergeOptions from './utils/merge-options.js';
 import document from 'global/document';
 import * as browser from './utils/browser.js';
 
+const defaults = {
+  liveTolerance: 60
```

Comment:
I'm not fully sure if we should only set the default to 60s for Freely devices but even if we didn't that would be a change within the C5 player anyways and not the library.

Reply:
Are we planning to always override this default in the player?

Reply:
My understanding is that we're not going to be overriding the default in the player and will just be using 60s as the `liveTolerance` value

Reply:
Ok, the reason I ask is because having a value this large would make [this problem](https://github.com/videojs/video.js/pull/6405#issuecomment-574830915) which the videojs devs encountered very noticeable where a user could seek back in the video but it would still show as live. Then fixing that resulted in the rest of their pull request.
If we are ok with that user experience then this change is fine but if not then it would be a regression on other platforms.

Reply:
Ah yes that's a good point - I'm not sure whether having that user experience is something that we're ok with or not.

Reply:
Shall we chat in person quickly?